specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -555,6 +698,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet all government-issued active tropical storms. Information about the tropical storms includes, government ID, basin ID, year of origin, name and if it is subtropical.",
```

Comment:
Can we change this to 'Get Active Tropical Storms'? I think the name should be more explicit, especially when it's listed with all the other Weather Services once in production, we want users to know it's tropical storms not regular storms.

Reply:
Updated